include/ibm/locks.hpp

```diff
@@ -345,18 +278,15 @@ inline void Lock::releaseLock(const ListOfTransactionIds& refRids)
 
         else
         {
-            BMCWEB_LOG_DEBUG << "Removing the locks from the lock table "
+            BMCWEB_LOG_ERROR << "Removing the locks from the lock table "
```

Comment:
same here, this does not meet the scope of this change

Reply:
This change is to log a trace if there is a lock removal. We dont have the json file now, and there is nothing to track the lock activities at BMC. So moved these to the error log level to get them at production driver